src/backend/gl/gl_common.c

```diff
@@ -1213,6 +1213,14 @@ bool gl_last_render_time(backend_t *base, struct timespec *ts) {
 		return false;
 	}
 
+/// Obtain proc address for glGetQueryObjectui64v in OpenBSD
+/// because the symbol function not appear directly in libgl for Mesa in this OS
+#ifdef __OpenBSD__
```

Comment:
Do we need this `ifdef`?

Reply:
And we should save this pointer somewhere,  instead of calling GetProcAddress every frame.